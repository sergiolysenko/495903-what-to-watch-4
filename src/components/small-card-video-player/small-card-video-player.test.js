import React from "react";
import renderer from "react-test-renderer";
import SmallCardVideoPlayer from "./small-card-video-player.js";

const VideoPreview = {
  WIDTH: 200,
  HEIGHT: 200,
  IS_MUTED: false,
};

const movie = {
  preview: `https://archive.org/download/BigBuckBunny_124/Content/big_buck_bunny_720p_surround.mp4`,
  cardImg: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
};

it(`Render SmallCardVideoPlayer`, () => {

  const tree = renderer.create(
      <SmallCardVideoPlayer
        isMuted={VideoPreview.IS_MUTED}
        poster={movie.cardImg}
        source={movie.preview}
        width={VideoPreview.WIDTH}
        height={VideoPreview.HEIGHT}
        isPlaying={true}
      />, {
        createNodeMock: () => {
          return {};
        }
      }).toJSON();

  expect(tree).toMatchSnapshot();
});
