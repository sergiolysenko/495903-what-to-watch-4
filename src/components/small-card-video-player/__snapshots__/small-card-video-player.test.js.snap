// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render SmallCardVideoPlayer 1`] = `<video />`;
